packages/hardhat-plugin/src/ui/components/execution/ModuleParameters.tsx

```diff
@@ -0,0 +1,34 @@
+import type { ModuleParams } from "@ignored/ignition-core";
+import { Text, Newline } from "ink";
+
+export const ModuleParameters = ({
+  moduleParams,
+}: {
+  moduleParams?: ModuleParams;
+}) => {
+  if (moduleParams) {
+    const params = [];
+
+    for (const [key, value] of Object.entries(moduleParams)) {
```

Comment:
pipe the `.entries` into a map, to produce the `<Text></Text>` array. I wouldn't hold it against you if you did it inline on the template.